tests/common/test_utils.py

```diff
@@ -253,6 +254,8 @@ def construct_perf_analyzer_config(
         The name of the model
     output_file_name: str
         The name of the output file
+    output_llm_file_name: str
+        The name of the LLM output file
```

Comment:
I would rather call it export file than llm file since export file is not restricted to llm models.
```suggestion
    export_file_name: str
        The name of the profile export file
```